docs/pages/access-controls/access-graph.mdx

```diff
@@ -12,14 +12,61 @@ users, roles, and resources in your organization. It can help you answer questio
 - What users can access a specific resource?
 - What are the relationships between users, roles, and resources?
 
+## Getting started with Teleport Policy
+
 Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product that is only 
 available to Teleport Enterprise customers.
 
 After logging into the Teleport UI, go to the Management tab. If enabled, Access Graph options can be found
 under the Permission Management section.
 
+<Admonition type="note">
+Note: For Teleport Enterprise customers, Access Graph is enabled by default.
```

Comment:
```suggestion
Note: For managed Teleport Enterprise customers, Access Graph is enabled by default.
```

Since self-hosted Enterprise customers need to enable it